### PR TITLE
add msbuild16

### PR DESCRIPTION
**Pull request by patricklee2**

### CODE REVIEW

File: Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs [hunk 1]
```diff
@@ -99,6 +99,20 @@ internal override string ResolveNpmJsPath()
...
+        internal override string ResolveMSBuild16Dir()
+        {
+            string programFiles = SystemEnvironment.GetFolderPath(SystemEnvironment.SpecialFolder.ProgramFilesX86);
+            string[] probPaths = new[]{
+                Path.Combine(programFiles, "Microsoft Visual Studio", "2019", "Enterprise", "MSBuild", "Current", "Bin"), // visual studio Enterprise
+                Path.Combine(programFiles, "Microsoft Visual Studio", "2019", "Professional", "MSBuild", "Current", "Bin"), // visual studio Professional
+                Path.Combine(programFiles, "Microsoft Visual Studio", "2019", "Community", "MSBuild", "Current", "Bin"), // visual studio Community
+                Path.Combine(programFiles, "Microsoft Visual Studio", "2019", "BuildTools", "MSBuild", "Current", "Bin"), // msbuild tools
+                // above is for public kudu, below is for azure
+                Path.Combine(programFiles, "MSBuild-16.2.37902.0", "MSBuild", "Current", "Bin")
```
Comment (suwatch) [inline Kudu.Core/Infrastructure/PathUtils/PathWindowsUtility.cs]: Should still keep the 15 but lower than 16  - so we can RU in any order

**Comment by suwatch** [comment]
There are more changes than this.  We have to fix KuduScript repo to default to 16 and so on.

File: Kudu.Services.Web/updateNodeModules.cmd [hunk 2]
```diff
@@ -10,7 +10,7 @@ set counter=0
 set /a counter+=1
 echo Attempt %counter% out of %attempts%
 
-cmd /c npm install https://github.com/projectkudu/KuduScript/tarball/16de31b5f5ca590ea085979e5fa5e74bb62f647e
+cmd /c npm install https://github.com/projectkudu/KuduScript/tarball/a37efe5669a6f479d11cffeade3706cfdd67c358
 IF %ERRORLEVEL% NEQ 0 goto error
```
Comment (suwatch) [inline Kudu.Services.Web/updateNodeModules.cmd]: has the kuduscript change been merged to projectkudu/kuduscript?

Comment (patricklee2) [reply Kudu.Services.Web/updateNodeModules.cmd]: not yet, but the download still works

Comment (suwatch) [reply Kudu.Services.Web/updateNodeModules.cmd]: understood.   I am afraid if it did not get merged by any reason in kuduscript, we should not merge here either.   That 's why we should review/merge there first.  @ahmelsayed  could help with the review..

**Pull request closed without merging**